### DISCUSSION TITLE
userland and Ubuntu, code spaces?

### DISCUSSION THREAD

**Discussion by Canusmellthatbag** [sherlock-project/sherlock#2129] [General]
is it possible to install and run the project on android?

**Comment by sdushantha** [comment]
If you have Termux installed on your Android device, you should be able to install Sherlock on it. There a few YouTube videos demonstrating how to do it, so I'd suggest you to check them out

**Reply by Canusmellthatbag** [reply]
This site has been a struggle to navigate and remember how to do and where to look for things for me but I'm catching on finally, I'm still determined, believe it or not I still haven't successfully installed sherlock but I now have a laptop and I think today is my day. Oh yea I just now found this message as well 😂  I'm probably fucked
